src/internal/constants.ts

```diff
@@ -74,6 +74,7 @@ export const validOrigins = [
   'https://local.teams.live.com:8080', // local development
   'https://local.teams.office.com', // local development
   'https://local.teams.office.com:8080', // local development
+  'https://localhost:8080', // local development
```

Comment:
This is not a good idea. We used to have localhost in this list and removed it for security reasons.

Reply:
What was the security concern?